src/components/Select/Select.styles.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const SelectWrapper = styled.div`
+  padding: var(--small-space);
+  border: 0.1rem solid var(--logopink);
+`;
+
+export const SelectDropdown = styled.select`
+  display: block;
+  border: 0.1rem solid var(--lightblue);
+  border-radius: 4px;
+  background-color: var(--lightgrey);
+  color: var(--darkgrey);
+  padding: var(--smallest-space);
+  padding-left: 0;
+  width: 100%;
+  max-width: 500px;
+`;
```

Comment:
As mention in comments above a few changes to make here:

- You shouldn't use pixel values, we're using rem for sizing so change border-radius to `0.4rem`.
- Don't need a max-width, width, padding left or display.
- Nest the select in `SelectWrapper`.
- Create a dropdown arrow with the `::after` pseudo select (https://css-tricks.com/snippets/css/css-triangle/ great resource for creating triangles with css.
- Remove default dropdown arrow with `-moz-appearance: none;` and `-webkit-appearance: none;` 
- Position the items within the `SelectWrapper` with flexbox.
- Add correct padding values to select.

You should end up with something like this:
```
import styled from 'styled-components';

export const SelectWrapper = styled.div`
  display: flex;
  flex-direction: column;
  position: relative;

  select {
    font-size: 1.6rem;
    border: 0.1rem solid var(--lightblue);
    border-radius: 0.4rem;
    background-color: var(--lightgrey);
    color: var(--darkgrey);
    padding: var(--small-space) var(--smallest-space);
    -moz-appearance: none;
    -webkit-appearance: none;
  }

  &::after {
    content: ' ';
    display: block;
    position: absolute;
    top: 5.2rem;
    right: var(--smallest-space);
    width: 0;
    height: 0;
    border-left: 0.8rem solid transparent;
    border-right: 0.8rem solid transparent;
    border-top: 1.2rem solid var(--lightblue);
  }
`;
```
If you would like me to go through in more detail just let me know.